src/components/modals/CompareDialogsModal.tsx

```diff
@@ -157,6 +160,14 @@ class CompareDialogsModal extends React.Component<Props, ComponentState> {
         })
     }
 
+    @OF.autobind
+    onEdit() {
+        const validationResult = this.props.transcriptValidationResults[this.state.resultIndex]
+        const { history } = this.props
+        let url = `/home/${this.props.app.appId}/logDialogs?${DialogUtils.DialogQueryParams.id}=${validationResult.logDialogId}`
```

Comment:
What was the opinion on the other PR of generating these query strings from the object?

I think it's more scalable and less error prone because it eliminates typos, is easier to add a prop, etc. I had intended to clean up the whole client doing that.